### PR TITLE
Update "/borderless" to "/multi-currency-account" and rename "About Transferwise" copy

### PR DESCRIPTION
**Pull request by bud93411**
Updated `/:locale/borderless` to `/:locale/multi-currency-account`
Update `/:locale/borderless#borderless-explainer-video` and `/:locale/about` to  `/community/mission-and-philosophy`

### CODE REVIEW

File: items/navigation.json [hunk 1]
```diff
@@ -73,7 +73,7 @@
         },
         {
           "translationKey": "public-navigation.multi-currency-account.about",
-          "link": "/:locale/borderless#borderless-explainer-video"
+          "link": "/community/mission-and-philosophy"
```
Comment (twrdjpalmer) [inline items/navigation.json]: This doesn't seem right—the original link takes you to an explainer video of the borderless product, not "about transferwise". What doesn't work about the current link/changing it to the video section on the MCA page? Sounds like that needs fixing first and foremost

Comment (bud93411) [reply items/navigation.json]: As per our discussion in https://transferwise.slack.com/archives/GL9EMRYRL/p1584457381048300
shall we delete this? or any suggestions? 🙂

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+# v3.8.1
```
Comment (twrdjpalmer) [inline CHANGELOG.md]: Should be a minor change as we're changing behaviour

**Comment by KarlEST** [comment]
Does it mean we still route some of them to borderless page? Seems strange that we route some cases to `multi-currency-account` and other cases not

**Comment by bud93411** [comment]
> Does it mean we still route some of them to borderless page? Seems strange that we route some cases to `multi-currency-account` and other cases not

Yes. I had the same q and clarified/confirmed with Mike. We route some of the cases to `/borderless`

**Pull request merged**